static/js/message_list.js

```diff
@@ -673,6 +673,14 @@ $(document).on('message_selected.zulip zuliphashchange.zulip mousewheel', functi
     viewport.stop_auto_scrolling();
 });
 
+$(window).load(function () {
+    $('img.emoji').each(function () {
```

Comment:
Start a discussion on chat.zulip.org #design; I think that's a good place to figure this out :)